packages/commonwealth/test/integration/controllers/newProfiles.spec.ts

```diff
@@ -0,0 +1,122 @@
+import axios from 'axios';
```

Comment:
We need to be clear about whether this file is a unit or an integration test.

Reply:
I think it is a unit test. Basically this is only imported so we can stub the axios client in order to make the controller not need the backend in order to function.